README.md

```diff
@@ -17,31 +17,31 @@ opam user-setup install
 
 [opam-user-setup](https://github.com/OCamlPro/opam-user-setup) takes care of configuring Emacs and Vim to make best use of your current install. You can also [configure the editor](#editor-setup) yourself, if you prefer.
 
-Manually building and installing Merlin
+Manually Building and Installing Merlin
 =======================================
 
-Since version 4.0, merlin's repository has a dedicated branch per version of
-OCaml, and the branch name consist of the concatenation of OCaml major version
-and minor version. So, for instance, `OCaml 4.11.*` maps to branch `411`.
-The main branch is usually synchronized with the branch compatible with the
+Since version 4.0, Merlin's repository has a dedicated branch for each version of
+OCaml, and the branch name consists of the concatenation of OCaml major versions
+and minor versions. So, for instance, `OCaml 4.11.*` maps to branch `411`.
+The main branch is usually synchronised with the branch compatible with the
 latest (almost-)released version of OCaml.
 
-Note: if you're using an older version of OCaml (between 4.02 and 4.10) you will
+Note: if you're using an older version of OCaml (between 4.02 and 4.10), you will
 want to build the 3.4 branch, although it won't contain the most recent
 features.
 
 Compilation
 -----------
 
-Dependencies: ocamlfind, yojson >= 2.0.0, dune >= 2.7.
+Dependencies: OCamlFind, Yojson >= 2.0.0, Dune >= 2.7.
```

Comment:
```suggestion
Dependencies: `ocamlfind`, `yojson` >= 2.0.0, `dune` >= 2.7.
```

These refer to package names, they should follow the capitalization used for the package list:
https://opam.ocaml.org/packages/